state-chain/pallets/cf-broadcast/src/tests.rs

```diff
@@ -485,32 +409,34 @@ fn test_transmission_request_expiry() {
 		check_end_state();
 	});
 }
+
 // One particular case where this occurs is if the Polkadot Runtime upgrade occurs after we've
 // already signed a tx. In this case we know it will continue to fail if we keep rebroadcasting so
-// we should stop and rethreshold sign using the new runtime version.
+// we should stop and re-threshold sign using the new runtime version.
 #[test]
 fn re_request_threshold_signature_on_invalid_tx_params() {
 	new_test_ext().execute_with(|| {
 		let (_, apicall) = api_call(1);
 		let broadcast_id = initiate_and_sign_broadcast(&apicall, TxType::Normal);
-		let broadcast_attempt_id = BroadcastAttemptId { broadcast_id, attempt_count: 0 };
 
 		assert_eq!(
 			MockThresholdSigner::<MockEthereumChainCrypto, RuntimeCall>::signature_result(0),
 			AsyncResult::Void
 		);
-		assert!(AwaitingBroadcast::<Test, Instance1>::get(broadcast_attempt_id).is_some());
-		assert_eq!(BroadcastAttemptCount::<Test, Instance1>::get(broadcast_id), 0);
+		assert!(AwaitingBroadcast::<Test, Instance1>::get(broadcast_id).is_some());
+		assert_eq!(Broadcaster::attempt_count(broadcast_id), 0);
 
 		MockTransactionBuilder::<MockEthereum, RuntimeCall>::set_requires_refresh();
 
 		// If invalid on retry then we should re-threshold sign
 		Broadcaster::on_initialize(BROADCAST_EXPIRY_BLOCKS + 1);
+		Broadcaster::on_idle(BROADCAST_EXPIRY_BLOCKS + 1, 1_000_000_000_000_000u64.into());
```

Comment:
We should use `LARGE_EXCESS_WEIGHT` - but may not be required if we use the test utils